cli/scale.go

```diff
@@ -144,6 +159,64 @@ func runScale(args *docopt.Args, client *controller.Client) error {
 	return nil
 }
 
+func showFormations(client *controller.Client, releaseID string, showAll bool, app string) error {
+	release, err := determineRelease(client, releaseID, app)
+	if err != nil {
+		return err
+	}
+	var releases []*ct.Release
+	if showAll {
+		var err error
+		releases, err = client.AppReleaseList(app)
+		if err != nil {
+			return err
+		}
+	} else {
+		releases = []*ct.Release{release}
+	}
+
+	formations := make([]*ct.Formation, len(releases))
+	for i, r := range releases {
+		formation, err := client.GetFormation(app, r.ID)
+		if err == controller.ErrNotFound {
+			formation = &ct.Formation{
+				AppID:     app,
+				ReleaseID: r.ID,
+				Processes: make(map[string]int),
+			}
```

Comment:
I suppose we could make `formations` a `map[string]*ct.Formation` where the release id is the lookup key